fairlearn/datasets/_fetch_data.py

```diff
@@ -0,0 +1,91 @@
+import warnings
+from ._common import FairnessWarning
+
+from sklearn.datasets import fetch_openml
+
+
+def fetch_boston(*, version='1', cache=True, data_home=None,
+                 as_frame=False, return_X_y=False, warn=True):
+    """Load the boston housing dataset (regression).
+
+    Download it if necessary.
+
+    ==============   ==============
+    Samples total               506
+    Dimensionality               13
+    Features                   real
+    Target           real 5. - 50.
+    ==============   ==============
+
+    This dataset has known fairness issues. There's a "lower status of
+    population" (LSTAT) parameter that you need to look out for and a column
+    that is a derived from the proportion of people with a black skin color
+    that live in a neighborhood (B). This dataset should be used to explore
+    and benchmark fairness. Predicting a house-price while ignoring these
+    sensitive attributes is naive.
+
+    Parameters
+    ----------
+    version : integer or 'active', default='active'
+        Version of the dataset. Can only be provided if also ``name`` is given.
```

Comment:
AFAIK we either do `(name, version)` or `data_id`. This function calls `fetch_openml` with `data_id` which ignores the `version` (if I'm not mistaken).

Reply:
Will check.

Reply:
I've removed the version. Am using just `data_id` now.